cmd/skopeo/inspect.go

```diff
@@ -58,7 +59,7 @@ func inspectCmd(global *globalOptions) cli.Command {
 		Flags: append(append([]cli.Flag{
 			cli.BoolFlag{
 				Name:        "raw",
-				Usage:       "output raw manifest",
+				Usage:       "Output raw manifest",
```

Comment:
If not a separate PR, make this at least a separate commit, please.

Reply:
maybe in another PR.